runtime/testy/runtimeUtil.py

```diff
@@ -133,3 +135,7 @@ class TimeoutError(Exception):
 
 class RuntimeError(Exception):
   pass
+
+# Sensor type names are CamelCase, with the first letter capitalized as well
+config_file = open(os.path.join(os.path.dirname(__file__), '../../hibike/hibikeDevices.json'), 'r')
```

Comment:
I know this was a workaround to find the type of a motor, but this feels kind of jank to merge into master, i.e. we really shouldn't be creating this dictionary.